store/tikv/2pc_fail_test.go

```diff
@@ -33,6 +34,10 @@ func (s *testCommitterSuite) TestFailCommitPrimaryRpcErrors(c *C) {
 	err = t1.Commit(context.Background())
 	c.Assert(err, NotNil)
 	c.Assert(terror.ErrorEqual(err, terror.ErrResultUndetermined), IsTrue, Commentf("%s", errors.ErrorStack(err)))
+
+	// We don't need to call "Rollback" after "Commit" fails.
```

Comment:
The comment confuses me. Why do you call the `Rollback()` in line 39?

Reply:
This test is used to prove why the rollback was removed in "RunInNewTxn".